### PR TITLE
disconnect stuck connection on failover before retry

### PR DESCRIPTION
**Pull request by watsaqat**

### CODE REVIEW

**Comment by watsaqat** [comment]
reset

**Comment by watsaqat** [comment]
integration tests (draft, not tested):
https://github.com/scality/Integration/pull/949/files

File: eve/workers/pod.yml [hunk 1]
```diff
@@ -81,4 +126,4 @@ spec:
       path: /var/run/docker.sock
       type: Socket
   - name: artifacts
-    emptyDir: {}
+    emptyDir: {}
```
Comment (mrepop) [inline eve/workers/pod.yml]: Needs a newline at EOF

File: eve/workers/unit_and_feature_tests/redis/sentinel.conf [hunk 2]
```diff
@@ -32,4 +32,4 @@ dir /tmp
 #
 # Note: master name should not include special characters or spaces.
 # The valid charset is A-z 0-9 and the three characters ".-_".
-sentinel monitor scality-s3 127.0.0.1 6379 1
+sentinel monitor scality-s3 127.0.0.1 6379 1
```
Comment (mrepop) [inline eve/workers/unit_and_feature_tests/redis/sentinel.conf]: Needs a newline at EOF

**Review by tmacro — APPROVED** [review]

**Review by mrepop — APPROVED** [review]

**Pull request closed without merging**